src/core/pyversion.c

```diff
@@ -19,3 +23,30 @@ py_version_micro()
 {
   return PY_MICRO_VERSION;
 }
+
+#include <sys/utsname.h>
+
+#define STRINGIFY(s) #s
+#define STR(s) STRINGIFY(s)
+
+int
+__syscall_uname(intptr_t buf)
```

Comment:
Can you share the uname before and after this change? Is that the thing that gets displayed when you start the interpreter or something else? It might still be useful to display the emscripten version there no?

For instance (assuming I'm not mistaken about what this does), on Linux the interpreter startup string is,
```
Python 3.10.4 | packaged by conda-forge | (main, Mar 24 2022, 17:39:04) [GCC 10.3.0] on linux
```
so it includes the compiler version. In our currently we have,
```
Python 3.11.3 (main, Sep 25 2023 20:45:01) on WebAssembly/Emscripten
```
and IMO it could be useful to include both Emscripten and Pyodide versions there.

Maybe,
```
Python 3.11.3 (main, Sep 25 2023 20:45:01) packaged by Pyodide (ABI 20230801) [Emscripten 1.38.1] on wasm32
```
though it's a bit too long, hmm.

Reply:
It still says:
```
Python 3.11.3 (main, Oct 13 2023 21:46:59) on WebAssembly/Emscripten
```
there. I'm not actually sure where all the different things come from...

Reply:
__Before__

```
>>> import platform
>>> platform.uname()
uname_result(system='Emscripten', node='emscripten', release='3.1.45', version='#1', machine='wasm32')
```

__After__

```
>>> import platform
>>> platform.uname()
uname_result(system='Pyodide', node='pyodide', release='202308_0', version='#1', machine='wasm32')
```

We can probably update our console banner string:

https://github.com/pyodide/pyodide/blob/5a1826355afe38a448ea06e029c3836ba00308d1/src/py/pyodide/console.py#L26-L29

Reply:
Interestingly conda-forge patches platform.py to display the conda-forge banner: https://github.com/conda-forge/python-feedstock/blob/main/recipe/brand_python.py

Reply:
OK, makes sense